docker/php/php.ini

```diff
@@ -1,8 +0,0 @@
-session.auto_start = Off
-date.timezone = Europe/Paris
-php_zip.dll
-
-# http://symfony.com/doc/current/performance.html
-opcache.max_accelerated_files = 20000
-realpath_cache_size = 4096K
```

Comment:
I think you don't need to remove all the file but just this line

Reply:
Ok I'll try tonight